### ISSUE TITLE
is:ornamented and is:hasornament don't highlight transmog items

### ISSUE THREAD

**Issue by inexorableAce** [DestinyItemManager/DIM#6805]
**Describe the bug and how to make it happen**
Seems like actual eververse style ornaments and the new transmog unlocked ones are different enough that our is:ornamented and is:hasornament searches do not pick up transmog ornamented items. 
![image](https://user-images.githubusercontent.com/29002828/120095986-051a5900-c0f7-11eb-8597-a76b187d0120.png)

**Comment by gusdleon** [comment]
Thanks @inexorableAce :)
Following 👀

**Comment by bhollis** [comment]
Yeah, you'll notice that transmogged ornaments don't have the "ornament" background either. That's because those mods aren't actually ornaments - Bungie actually just uses the actual armor InventoryItem as the plug for the socket. Thus they aren't detectable as an ornament per se. To put it another way, there isn't a separate "ornament" copy of every item.
